04-working-with-build-artifacts/readme.md

```diff
@@ -56,18 +56,17 @@ jobs:
 
 We're creating an artifact `dependencies`, and we're grabbing the content for this artifact from `node_modules`
 
-Now we need the simmetric operation [Download a Build Artifact](https://github.com/marketplace/actions/download-a-build-artifact)
-
+Now we need the symmetric operation: [Download a Build Artifact](https://github.com/marketplace/actions/download-a-build-artifact)
 
 * Update `ci.yml`
 
-```yml
+```diff
```

Comment:
I don't like to use diff with yaml. Why? Because identation mess things up, instead we can use a comment. Maybe it's a good time to start a demo writing style guide.

Reply:
Updated